src/components/Capture/Document.js

```diff
@@ -59,22 +61,39 @@ class Document extends Component {
     } = this.props
     const copyNamespace = `capture.${isPoA ? poaDocumentType : documentType}.${side}`
     const title = translate(`${copyNamespace}.title`)
-    const moreProps = { ...this.props, onError: this.handleError }
-
-    return useWebcam && hasCamera ?
-      <DocumentAutoCapture
-        { ...moreProps }
-        renderTitle={ <PageTitle { ...{ title, subTitle } } smaller /> }
-        renderFallback={ isDesktop ? this.renderCrossDeviceFallback : this.renderUploadFallback }
-        containerClassName={ style.documentContainer }
-        onValidCapture={ this.handleCapture }
-      /> :
+    const propsWithErrorHandling = { ...this.props, onError: this.handleError }
+    const renderTitle = <PageTitle {...{title, subTitle}} smaller />
+    const renderFallback = isDesktop ? this.renderCrossDeviceFallback : this.renderUploadFallback
+    if (useWebcam && hasCamera) {
+      return (
+        <DocumentAutoCapture
+          {...propsWithErrorHandling}
+          renderTitle={ renderTitle }
+          renderFallback={ renderFallback }
+          containerClassName={ style.documentContainer }
+          onValidCapture={ this.handleCapture }
+        />
+      )
+    } else if (useLiveDocumentCapture && hasCamera) {
```

Comment:
Minor: I personally find it more readable to use `if` when the previous condition has a return